### PR TITLE
Use the same derivation path validation as the SDK

### LINKED ISSUE #4665: Use the same derivation path validation as the SDK
### Description

Use the same derivation path validation as the SDK.

### Motivation

To not manage validation in two places. It is better with one source of truth. 

### Additional Information

Modify `getDerivationPathErrorMessage` to use cryptography.utils.parseKeyDerivationPath();

https://github.com/LiskHQ/lisk-sdk/issues/7877

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #4665 and #4600

### How was it solved?

- Use `cryptography.utils.parseKeyDerivationPath`
- Updated menu items as designed

### How was it tested?

- Updated unit tests
- Test the menu items visual and compare with design

### CODE REVIEW

**Review by oskarleonard — DISMISSED** [review]

File: src/modules/wallet/utils/account.js [hunk 1]
```diff
@@ -312,37 +312,12 @@ export const validate2ndPass = async (account, passphrase, error) => {
  * @returns {string} - Error message
  */
 export const getDerivationPathErrorMessage = (derivationPath) => {
-  if ((!derivationPath || !derivationPath.startsWith('m')) || !derivationPath.includes('/')) {
```
Comment (soroushm) [inline src/modules/wallet/utils/account.js:315]: validation is missing now, i think we should keep the validation

Comment (oskarleonard) [reply src/modules/wallet/utils/account.js:315]: We still have the validation, `getDerivationPathErrorMessage` now makes use of the SDK function `parseKeyDerivationPath` to validate. To make sure we are using the same validation as the SDK

File: src/modules/account/components/EditAccount/EditAccount.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ const EditAccount = ({ history }) => {
       <EditAccountForm onBack={history.goBack} />
       <SetPasswordSuccess
         encryptedPhrase={currentAccount}
-        headerText={t('Edit name successful')}
+        headerText={t('Edit account name successful')}
```
Comment (oskarleonard) [inline src/modules/account/components/EditAccount/EditAccount.js:20]: Shouldnt we commit this change in common.json file as well?

File: src/modules/wallet/utils/account.js (hunk 1 above)
Comment (soroushm) [reply src/modules/wallet/utils/account.js:315]: in this case this error is message is wrong maybe we can have it as `"derivationPath not found"`

**Review by oskarleonard — DISMISSED** [review]

**Review by soroushm — DISMISSED** [review]

**Review by oskarleonard — APPROVED** [review]

**Pull request merged**